.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,11 @@
+name: Run ShellCheck on PR
+on: [pull_request]
+
+jobs:
+  shellcheck:
+    name: Shellcheck
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Minor, `actions/checkout@v3` would be better, [1]

[1] https://github.com/actions/checkout

Reply:
Thanks, updated